config/ompi_fortran_get_alignment.m4

```diff
@@ -132,3 +132,155 @@ EOF
     AS_VAR_POPDEF([type_var])dnl
     OPAL_VAR_SCOPE_POP
 ])
+
+# OMPI_FORTRAN_F08_GET_HANDLE_ALIGNMENT(type, variable to set)
+# ------------------------------------------
+AC_DEFUN([OMPI_FORTRAN_F08_GET_HANDLE_ALIGNMENT],[
+    # Use of m4_translit suggested by Eric Blake:
+    # http://lists.gnu.org/archive/html/bug-autoconf/2010-10/msg00016.html
+    AS_VAR_PUSHDEF([type_var],
+       m4_translit([[ompi_cv_fortran_alignment_$1]], [*], [p]))
+
+    AC_CACHE_CHECK([alignment of Fortran $1], type_var,
+        [OMPI_FORTRAN_MAKE_C_FUNCTION([ompi_ac_alignment_fn], [alignment])
```

Comment:
For an F08 test, we should probably just use a proper BIND(C) routine, rather than figuring out the symbol munging style of the Fortran compiler.  I.e., just BIND(C) to the C function name.